lib/utils/ensure-single-browser-tab-only.js

```diff
@@ -0,0 +1,78 @@
+import React from 'react';
+import ReactDOM from 'react-dom';
+
+import BootWarning from '../components/boot-warning';
+
+const HEARTBEAT_DELAY = 1000;
+const clientId = uuidv4();
+console.log(clientId);
+const emptyLock = [null, -Infinity];
+const foundElectron = window.process && window.process.type;
+
+if (!foundElectron) {
+  const sesh = grabSessionLock();
```

Comment:
can we use something less abbreviated than `sesh`? I'm afraid this might be confusing in the code

 - `lockAttempt`
 - `lockStatus`
 - `sessionLock`
 - something better